tests/app/hooks/apiRequest.test.tsx

```diff
@@ -0,0 +1,113 @@
+import { ApiOptions, apiRequest } from '../../../src/hooks/apiRequest';
+
+global.fetch = jest.fn();
+
+describe('apiRequest', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should use default options for a basic GET request', async () => {
+    const mockResponse = { data: 'test data' };
+    (global.fetch as jest.Mock).mockResolvedValueOnce({
+      ok: true,
+      json: jest.fn().mockResolvedValueOnce(mockResponse),
+    });
+
+    const response = await apiRequest('/test-endpoint');
+
+    const fetchCalls = (global.fetch as jest.Mock).mock.calls;
+
+    expect(fetchCalls[0][0]).toBe(
+      `${process.env.NEXT_PUBLIC_BACKEND_URL}/test-endpoint`
+    );
+    expect(fetchCalls[0][1]).toMatchObject({
+      method: undefined,
+      headers: { 'Content-Type': 'application/json' },
```

Comment:
Update this wrt the update to `DEFAULT_OPTIONS`.

```suggestion
      method: 'GET',
```